### PR TITLE
Updates to OWNERS file to add @margarethaley as a reviewer

### PR DESCRIPTION
**Pull request by quaid**

### CODE REVIEW

**Comment by quaid** [comment]
I'm going to retest the pre-commit, but here is literally everything I did to make the patch and it included a pre-commit pass:

```[kwade@toolbox community]$ git branch quaid-OWNERS-01
[kwade@toolbox community]$ git checkout quaid-OWNERS-01
Switched to branch 'quaid-OWNERS-01'
[kwade@toolbox community]$ vi OWNERS 
[kwade@toolbox community]$ git add OWNERS 
[kwade@toolbox community]$ git commit OWNERS
Tabs remover.............................................................Passed
Trim Trailing Whitespace.................................................Passed
Check for merge conflicts................................................Passed
Fix End of Files.........................................................Passed
Check for added large files..............................................Passed
Check for case conflicts.................................................Passed
Check JSON...........................................(no files to check)Skipped
Check for broken symlinks............................(no files to check)Skipped
Detect Private Key.......................................................Passed
yamllint.............................................(no files to check)Skipped
[quaid-OWNERS-01 b189b0d] Adding @margarethaley to list of reviewers - This list seems to be the one used by the bot in randomly assinging reviewers
 1 file changed, 1 insertion(+)
[kwade@toolbox community]$ git push --set-upstream origin quaid-OWNERS-01 
Enumerating objects: 5, done.
Counting objects: 100% (5/5), done.
Delta compression using up to 4 threads
Compressing objects: 100% (3/3), done.
Writing objects: 100% (3/3), 407 bytes | 203.00 KiB/s, done.
Total 3 (delta 2), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (2/2), completed with 2 local objects.
remote: 
remote: Create a pull request for 'quaid-OWNERS-01' on GitHub by visiting:
remote:      https://github.com/operate-first/community/pull/new/quaid-OWNERS-01
remote: 
To github.com:operate-first/community.git
 * [new branch]      quaid-OWNERS-01 -> quaid-OWNERS-01
Branch 'quaid-OWNERS-01' set up to track remote branch 'quaid-OWNERS-01' from 'origin'.
```
/retest

**Comment by quaid** [comment]
Tagging @hemajv & @harshad16 : this is a small commit but I'm doing something incorrect. The `pre-commit` check works fine locally.

File: README.md [hunk 1]
```diff
@@ -40,3 +81,4 @@ There are a new set of labels available project wide:
 
  * A PR that signals the completion of an important Epic worthy of making some noise about and including in a newsletter can be labeled `area/community`, `kind/news`, and `kind/marketing`.
  * An issue is related to the experience of users joining the project to become contributors is labeled `area/community`, `area/user`, `area/contributor`, `kind/experience`, and `kind/onboarding`.
+ * A write-up on how to setup a clone of a repository for doing work intended to be contributed to the community would be labeled `area/contributor`, `kind/documentation`, `kind/onboarding`, and `kind/handbook`.
```
Comment (HumairAK) [inline README.md:84]: @quaid the lint is failing here, you need to remove the trailing white space after this line, and also add a newline (with no whitespace). 

If you run `pre-commit run --all-files` from the root of this commit, it should automatically fix these errors for you

Comment (quaid) [reply README.md:84]: Oh! This is unhappy with the other PR, that's part of the mystery to me.

I'm going to write out my understanding to make sure I'm not missing something:
* If I forget to run `pre-commit install` on a new repo I've cloned, it won't run `pre-commit` locally and I will get two failed checks—one for not having a pre-commit environment running, the other for any failures of the actual commit itself in check.
* If I then run `pre-commit install`, the only way the bit knows and can test if I have done that is if I commit any change to the repo so it shows the change to my local installation. Is that accurate?
* If there are any files that fail pre-commit normally, I get to fix them before the commit. If I commit files without pre-commit and there _are_ errors in the files, then running pre-commit subsequently will catch and fix those files. I now have actual changes to those files, so restaging and commiting them is what I would do anyway.

Thanks!

**Review by oindrillac — APPROVED** [review]
/lgtm
/approve

**Review by margarethaley — APPROVED** [review]

**Pull request merged**